src/test/java/com/camacho/simpleannotation/builders/FakeClassNoDefaultConstructor.java

```diff
@@ -58,4 +58,17 @@ public Object getObject() {
     public void setObject(Object object) {
         this.object = object;
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        FakeClassNoDefaultConstructor that = (FakeClassNoDefaultConstructor) o;
+        return i == that.i && Double.compare(that.d, d) == 0 && Objects.equals(s, that.s) && Objects.equals(strings, that.strings) && Objects.equals(object, that.object);
```

Comment:
This line is too long. If code will be presented, it would be easier to see the hole line without side scrolling if you line break conditions here.

Reply:
Done. Revised in [pull request 8](https://github.com/OscarCamacho/simpleannotation/pull/8)